### PR TITLE
Tweak pieces regarding community-language support

### PR DESCRIPTION
**Pull request by BanzaiMan**
`travis-core` is no longer relevant, so we remove that bit.

### CODE REVIEW

File: user/languages/community-supported-languages.md [hunk 1]
```diff
@@ -85,6 +73,15 @@ realize support for builds in a new language.
    add your language to the `app/utils/keys-map.coffee` file and submit a
    pull request for this change.
 
+1. Other bits
```
Comment (plaindocs) [inline user/languages/community-supported-languages.md]: Maybe something like 

> 1. If you want to support build matrix expansion for various language versions (e.g., Ruby 2.2, 2.1, etc.), please coordinate with the Travis CI team to find out exactly what is required.

**Comment by plaindocs** [comment]
I've only reviewed the changes, do you want me to look at the rest as well? It is probably due an update.

**Comment by BanzaiMan** [comment]
@plaindocs Yes. Please have a look over.

**Comment by plaindocs** [comment]
See #833

**Pull request merged**